subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt

```diff
@@ -200,6 +200,34 @@ abstract class InHouseInstrumentationTestRunner :
         }
     }
 
+    private fun injectTestMetadata(arguments: Bundle) {
```

Comment:
it looks like other (new) class responsibility which should be delegated to testrunner

Reply:
Will fix in another PR

Reply:
https://github.com/avito-tech/avito-android/pull/704